README.md

```diff
@@ -21,6 +21,34 @@ import '@github/typing-effect-element'
 </typing-effect>
 ```
 
+## Accessibility
+
+This component detects whether `prefers-reduced-motion` is set on the window:
+
+```js
+window.matchMedia('(prefers-reduced-motion)').matches === true
+```
+
+If this evaluates to true, any content lines provided will be appended immediately rather than being typed out with a delay.
+
+The data attribute `data-reduced-motion` can be used to override the window media value.
```

Comment:
What do you think about not including the option to overwrite? I can't think of a scenario where we'd want to ignore the reduced motion request.

Reply:
I was conflicted about adding this as well. I guess my thinking was that in the case this component was already being used in different locations, then this could be an escape hatch for programmatically determining the animated behavior to match up with previous behaviors. But seeing as this is only currently being used in one low-traffic site, I would be okay with removing this altogether. ¯\_(ツ)_/¯ Up to you all!

Reply:
I'd be down to remove the data attribute for now. I think it would be easier to add later if requested than remove it later.